packages/devtools_app/lib/src/inspector/flutter/layout_tab.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import '../../inspector/inspector_text_styles.dart' as inspector_text_styles;
+import '../diagnostics_node.dart';
+import '../inspector_tree.dart';
+
+class InspectorDetailsTabController extends StatelessWidget {
+  const InspectorDetailsTabController({this.detailsTree, Key key})
+    : super(key: key);
+
+  final Widget detailsTree;
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 2,
+      child: Column(
+        children: <Widget>[
+          const TabBar(
+            tabs: [
+              Tab(text: 'Details Tree'),
```

Comment:
can we make these tabs not take as much horizontal space?
right now the Layout Details tab looks aligned with the expand-collapse buttons which looks a bit confusing.

Reply:
I agree, will look into this tomorrow, there should be a way